src/FordhamBank/Enums/TransactionType.java

```diff
@@ -0,0 +1,9 @@
+package FordhamBank.Enums;
+
+public enum TransactionType {
```

Comment:
We're not doing CHARGE or ONLINE yet as we dont have a way of actually knowing a user has "bought" something. Keep the first 3 only for now